//test error

int main(){
    int x(){
        return 4;
    }

    return x();
}